code/controllers/subsystems/processing/mobs.dm

```diff
@@ -8,6 +8,16 @@ PROCESSING_SUBSYSTEM_DEF(mobs)
 	process_proc = /mob/proc/Life
 
 	var/list/mob_list
+	var/list/mob_living_by_zlevel[][]
 
 /datum/controller/subsystem/processing/mobs/PreInit()
 	mob_list = processing // Simply setups a more recognizable var name than "processing"
+	MaxZChanged()
+
+/datum/controller/subsystem/processing/mobs/proc/MaxZChanged()
+	if(!islist(mob_living_by_zlevel))
+		mob_living_by_zlevel = new /list(world.maxz, 0)
```

Comment:
What is the second oargument on the list for?

Reply:
Creates a multidimensional list consisting of `world.maxz` lists, each containing 0 elements